From 2089b514045d2de64a5d9c54e241731e85d77df2 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 14 Dec 2020 22:11:59 -0800
Subject: [PATCH] makefile.am: Fix build when build dir is not same as
 sourcedir

This ensures right include paths are added to compiler

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index eba85a1..da998ff 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -19,7 +19,7 @@ if COND_DEBUG
 MAYBE_DEBUG=-g -O0
 endif
 
-AM_CFLAGS = $(MAYBE_COVERAGE) $(MAYBE_DEBUG) $(MAYBE_TRACE)
+AM_CFLAGS = $(MAYBE_COVERAGE) $(MAYBE_DEBUG) $(MAYBE_TRACE) -I$(srcdir)/inc
 
 check_PROGRAMS = tests/test-shunting-yard
 tests_test_shunting_yard_SOURCES = src/shunting-yard.c inc/shunting-yard.h \
-- 
2.29.2

